app/ts/Elicitation/Interface/IPreciseSwingAnswer.ts

```diff
@@ -0,0 +1,4 @@
+export default interface IPreciseSwingAnswer {
+  criterionId: string;
+  importance: number;
+}
```

Comment:
Do we actually need these answer interfaces? Can't we just store them in the context? Aren't they a leftover from when we were actually storing them in de database in the survey?

Reply:
Deleted